### DISCUSSION TITLE
More powerful pets

### DISCUSSION THREAD

**Discussion by avallet1** [DiscordDungeons/Bugs#991] [Ideas]
At this moment, pet Rock is the most powerful pet we can find and as we start with it, **#!catch**  is useless.
Moreover, pet Rock may be the most powerful pet but it's still lacking. After a certain level, it adds less damages than a beer and it's just bothersome to have to heal it.
I think more powerful pets should be added.

**Comment by Feathers8** [comment]
Lol they aren't adding more powerful pets after nerfing them (they used to be so op, weapons were basically useless)
I think it's intended that most of the damage should be coming from the character

**Reply by Sirfirelord69** [reply]
Actually, the Innocent Creature is the most powerful pet. you just have to catch it and the way to do it is incredibly complicated
